src/index.html

```diff
@@ -1,24 +1,24 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Output Management</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-  <body>
-    <div class="container">
-      <div class="list">
-        <h2>To-do list</h2>
-        <div class="row">
-          <input type="text" id="input-box" placeholder="add to your list" />
-          <button class="add-btn">Add</button>
-        </div>
-        <ul id="list-container"></ul>
-        <button class="removeButton">clear</button>
-      </div>
-    </div>
-    <script type="module" src="index.js"></script>
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Output Management</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="list">
+        <h2>To-do list</h2>
+        <div class="row">
+          <input type="text" id="input-box" placeholder="add to your list" />
+          <button class="add-btn">Add</button>
+        </div>
+        <ul id="list-container"></ul>

```

Comment:
- @LethaboMot While editing a task I noticed I can edit the `x` symbol intended for deleting the task, which makes it impossible to delete that task if I remove it, It should not be presented in that way of allowing me to delete a marker for a functionality. Below is the video that shows this scenario. 


-  **Video**
 
[Screencast from 05-13-2023 05:18:56 PM.webm](https://github.com/LethaboMot/weppack2/assets/58771507/58e051df-7a13-4ff0-ae64-903d479b6dfc)